react/components/InputCurrency/README.md

```diff
@@ -22,8 +22,8 @@ initialState = { value: undefined }
       placeholder="Type a monetary value"
       locale="pt-BR"
```

Comment:
Should we use `pt-BR` and `BRL` since we are enforcing English as the main language? 🤔

Reply:
good point

Reply:
Good point, updated.